src/Router.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import Home from './containers/Home';
-
+import Home from './containers/Home/Home';
```

Comment:
```suggestion
import Home from './containers/Home';
```
This should be like that, in inside `./Home/index.js` should import Home.js and export default it